### PR TITLE
UG ped, DG intro, diagrams, use cases

### LINKED ISSUE #190: [PE-D] Minor Bug with Table of Contents
The link in the Table of Contents for `5.2 Fridge-related Commands` doesn't work.

-------------
Labels: `severity.Low` `type.DocumentationBug`
original: lowjiefeng1998/ped#1

### LINKED ISSUE #191: [PE-D] Trailing <br>s in the Section 5 of the UG
There are some trailing <br in Section 5 of the UG.

-------------
Labels: `severity.VeryLow` `type.DocumentationBug`
original: lowjiefeng1998/ped#2

### LINKED ISSUE #194: [PE-D] Hard to read documentation in Section 2.3 Bullet Point 1
This isn't a critical bug, more of an area to improve/that can be clearer :)

This is a bit confusing. You can try this from kramdown:
https://kramdown.gettalong.org/converter/html.html#showing-whitespace-in-a-code-block

-------------
Labels: `severity.Low` `type.DocumentationBug`
original: lowjiefeng1998/ped#6

### LINKED ISSUE #196: [PE-D] Unclear message in editF
This sample command in the help text doesn't work for `editF`:

-------------
Labels: `severity.Low` `type.FunctionalityBug`
original: lowjiefeng1998/ped#8

### LINKED ISSUE #198: [PE-D] Typo in UG for the recommend command
-------------
Labels: `severity.Low` `type.DocumentationBug`
original: lowjiefeng1998/ped#11

### LINKED ISSUE #210: [PE-D] Rogue <br>
No details provided.

-------------
Labels: `severity.Low` `type.DocumentationBug`
original: rtshkmr/ped#3

### LINKED ISSUE #218: [PE-D] Inaccuracies in user guide
The <br> statement was not correctly inserted.

-------------
Labels: `severity.Low` `type.DocumentationBug`
original: jonasngs/ped#6

### LINKED ISSUE #224: [PE-D] Documentation bug with instruction tag
The UG conflicts with the error message of the app. Instr in app. Inst in UG.

-------------
Labels: `severity.High` `type.DocumentationBug`
original: todoge/ped#5

### LINKED ISSUE #228: [PE-D] Possible error in UG
Not sure what "get editF" command is.

-------------
Labels: `severity.VeryLow` `type.DocumentationBug`
original: todoge/ped#9

### LINKED ISSUE #230: [PE-D] Can help target audience better
Target audience is meant for busy office workers with little time.
Users have to manually update each ingredient in their fridge. It would make more sense for busy office worker to execute "eatR" then have the quantities of each ingredient in the fridge editted automatically since they would not want to spend so much time managing the app because they are busy.

-------------
Labels: `severity.Medium` `type.FeatureFlaw`
original: todoge/ped#11

### PR DESCRIPTION
**Pull request by oliviajohansen**

### CODE REVIEW

**Review by caitlinjee — APPROVED** [review]
lgtm

**Review by daongochieu2810 — APPROVED** [review]

File: docs/DeveloperGuide.md [hunk 1]
```diff
@@ -514,7 +561,7 @@ Given below is an example usage scenario and how the mechanism behaves:
 
 #### 3.6.2 Design Considerations (**Get Edit Recipe**) <a id="362-design-consideration-get-edit-recipe"></a>
 ##### Aspect: Concern while adding a new feature <a id="3621-aspect"></a>
-* Workflow must be consistent with other get edit commands.
+* Workflow must be consistent with other get edit commands e.g get ingredient command.
```
Comment (TianYong-Goh) [inline docs/DeveloperGuide.md]: I feel the eg is a bit extra cause we only have another one get edit command

**Review by TianYong-Goh — APPROVED** [review]
lgtm

File: docs/DeveloperGuide.md (hunk 1 above)
Comment (oliviajohansen) [reply docs/DeveloperGuide.md]: o ya ok

**Review by jiaqi20 — APPROVED** [review]
lgtm

**Pull request merged**